src/Akka.Persistence.Sql/Query/SqlReadJournal.cs

```diff
@@ -68,7 +67,7 @@ public class SqlReadJournal :
             _mat = Materializer.CreateSystemMaterializer(
                 context: system,
                 settings: ActorMaterializerSettings.Create(system),
-                namePrefix: $"l2db-query-mat{configPath}");
+                namePrefix: $"l2db-query-mat-{Guid.NewGuid():N}");
```

Comment:
What was the fix here?

Reply:
That was not the fix, the `SqlReadJournal` is a bit different in that it creates a materializer, which none of our other read journal does.

It tries to create a single materializer by appending the "config path".

The problem is that "config path" were never being passed into the provider, the provider was created through reflection magic and none of the activator code passes in the "config path".

So I just remove that ambiguous code and replace it with a GUID